build_projects/dotnet-host-build/build.sh

```diff
@@ -85,7 +85,9 @@ done < "$REPOROOT/branchinfo.txt"
 [ -z "$DOTNET_INSTALL_DIR" ] && export DOTNET_INSTALL_DIR=$REPOROOT/.dotnet_stage0/$(uname)
 [ -d "$DOTNET_INSTALL_DIR" ] || mkdir -p $DOTNET_INSTALL_DIR
 
-$REPOROOT/scripts/obtain/dotnet-install.sh --channel $CHANNEL --verbose
+
+DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/b939c88e0cf7c2e204f91a5161acdd2f5a732bf9/scripts/obtain/dotnet-install.sh"
```

Comment:
But we are taking the actual latest CLI right?  It doesn't really make sense to use a hard-coded install script version to install the latest CLI.  I would think they should be either both hard-coded versions or both "latest".

Another thought:  the BuildTools stuff needs a CLI as well.  Currently in the CLI repo, we download 2 versions: the BuildTools version, and the "latest".  Maybe we should change this to only require 1 version of the CLI.  i.e. use the BuildTools version for both BuildTools and for running the build.